src/libraries/shims/ApiCompatBaseline.PreviousNetCoreApp.txt

```diff
@@ -0,0 +1,51 @@
+Compat issues with assembly mscorlib:
+CannotRemoveAttribute : Attribute 'System.CLSCompliantAttribute' exists on 'System.TypedReference.SetTypedReference(System.TypedReference, System.Object)' in the contract but not the implementation.
```

Comment:
NIT: Should we add these attributes to APICompat list of ignored attributes so we don't get them as diffs? I only mention it because if we have a clean baseline, it will be easier to keep it like that.

Reply:
I've thought about it, there's been a bit of tension about which attributes we ignore.  I think we need an independent discussion to drive consensus on ignoring these.

Reply:
So CLSCompliant is noisy.  For that I've opened https://github.com/dotnet/runtime/issues/33316

The rest were introduced with https://github.com/dotnet/corefx/pull/42427 when I turned on attribute validation on members.  These are "noticeable" differences but acceptable.